lib/flutter_barcode_listener.dart

```diff
@@ -67,6 +67,8 @@ class _BarcodeKeyboardListenerState extends State<BarcodeKeyboardListener> {
 
   final bool _useKeyDownEvent;
 
+  bool _isShiftPressed = false;
```

Comment:
Why not move variable declaration closer to implementation, on line 117 perhaps?

Reply:
It can't, because when i put there, it will always be recreated. And it will make a dead code. 
<img width="779" alt="image" src="https://github.com/shaxxx/flutter_barcode_listener/assets/25898509/15b35ebd-9b1e-4a77-9427-e140cea113a9">